ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -51,6 +51,15 @@ do
   cp "${item}" "${dir}/manifests/${manifest##manifest_}"
 done <   <( find "${SHARED_DIR}" -name "manifest_*.yml" -print0)
 
+if [[ "${USE_ETCD_RAMDISK}" == "true" ]]; then
+  openshift-install --dir=${dir} create ignition-configs
+  python -c \
+      'import json, sys; j = json.load(sys.stdin); j[u"systemd"] = {}; j[u"systemd"][u"units"] = [{u"contents": "[Unit]\nDescription=Mount etcd as a ramdisk\nBefore=local-fs.target\n[Mount]\n What=none\nWhere=/var/lib/etcd\nType=tmpfs\nOptions=size=2G\n[Install]\nWantedBy=local-fs.target", u"enabled": True, u"name":u"var-lib-etcd.mount"}]; json.dump(j, sys.stdout)' \
```

Comment:
Sorry that I didn't think about this before. This change to the master stub ignition is going to get converted into a MachineConfig by the installer [1]. It is recommended that you just wrote the MachineConfig in a manifest directly instead of modifying the stub ignition.

[1] https://github.com/openshift/installer/blob/master/pkg/asset/ignition/machine/master_ignition_customizations.go

Reply:
looking into this.